kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -9,22 +9,25 @@ ssui_root="$repos_root/manageiq-ui-self_service"
 
 rm -rf $app_root
 
-git clone <%= @appliance_checkout.remote %> $appliance_root
+git clone <%= "--separate-git-dir /tmp/appliance.git" if @build_type == "release" %> \
+        <%= @appliance_checkout.remote %> $appliance_root
```

Comment:
@Fryguy @bdunne @jrafanie So it appears that we can't easily remove the .git directory. Building on @bdunne's suggestion, the following appears to work:

```
  git clone --depth 1 --branch <branch_or_tag> https://github.com/manageiq/manageiq.git
  git config remote.origin.fetch '+refs/heads/*:refs/remote/origin/*'
  git config branch.<branch>.remote origin
  git config branch.<branch>.merge refs/heads/<branch>
  git checkout -b <branch>
```

As far as I can see the result is exactly similar to the current situation (e.g. `git pull` will do the right thing automatically). The difference is about 100MB which is a good reduction.